src/idpi/operators/pot_vortic.py

```diff
@@ -57,8 +57,13 @@ def fpotvortic(
     """
     # target coordinates
     deg2rad = np.pi / 180
-    lat = (rho_tot["latitude"] * deg2rad).astype(np.float32)
-    rlat = get_rotated_latitude(w)
+    lat = (rho_tot["lat"] * deg2rad).astype(np.float32)
+
+    geo = w.attrs["geography"]
+    ny = geo["Nj"]
+    lat_min = geo["latitudeOfFirstGridPointInDegrees"]
```

Comment:
this is an example of information that I think could come from a grid object that is passed to operators that need geographical information, that is constructed at setup.

Reply:
Implementing that grid description object is not within the scope of this PR I believe.

Reply:
yes, I agree